### PR TITLE
Admission trial project files

### PR DESCRIPTION
**Pull request by yaseerokino**
- Added files from Microverse admission project trials

### CODE REVIEW

**Comment by yaseerokino** [comment]
- From your PR it seems you have some linter checks that were not successful, Kindly check your code and make the necessary changes to fix these errors

Screenshot 2021-07-07 at 12 39 38

Happy Coding !!!

**Review by yaseerokino — COMMENTED** [review]
#Changes required ♻️ 

- Please check all my comments and make all required changes 
- Submit for a re-review when you are done with all the required changes

Warm Regards and Happy Coding 😃

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            <div class="image"></div>
+            <h6>
+              <a
+                href="https://github.com/yaseerokino/yaseer-kossi"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Portfolio 1</a
+              >
+            </h6>
+          </div>
```
Comment (Temmarie) [inline index.html:37]: - [ ] It'll be nice if you could use semantic HTML tags for your project as this helps improve webpage accessibility.

_This is an example of an inline comment coming after the review summary._

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @yaseerokino  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

**Optional**
- [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:

This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: index.html (hunk 1 above)
Comment (yaseerokino) [reply index.html:37]: Thank you for the tip, I will make sure to use it that way in future projects

**Pull request merged**